src/components/ClearRefinements/ClearRefinements.tsx

```diff
@@ -0,0 +1,51 @@
+/** @jsx h */
+
+import { h } from 'preact';
+import cx from 'classnames';
+import Template from '../Template/Template';
+import { ClearRefinementsRenderState } from '../../connectors/clear-refinements/connectClearRefinements';
+import {
+  ClearRefinementsCSSClasses,
+  ClearRefinementsTemplates,
+} from '../../widgets/clear-refinements/clear-refinements';
+
+type ClearRefinementsComponentCSSClasses = Required<
+  {
+    [key in keyof ClearRefinementsCSSClasses]: string;
+  }
+>;
```

Comment:
how is this different from just

```ts
type ClearRefinementsComponentCSSClasses =  Required<ClearRefinementsCSSClasses>
```

Reply:
one is a string, the other is a string | string[], these get made more consistent in #4774